appOPHD/States/MainMenuState.cpp

```diff
@@ -26,7 +26,8 @@ MainMenuState::MainMenuState() :
 	}},
 	lblVersion{constants::Version},
 	mReturnState{this},
-	mFade{{this, &MainMenuState::onFadeComplete}}
+	mFade{{this, &MainMenuState::onFadeComplete}},
+	mFileIoDialog{{this, &MainMenuState::onLoadGame}}
```

Comment:
When connecting using the `Delegate`, we should remove the `connect` call using the `Signal`.

Edit: Which is in the following commit.